torchao/prototype/autoround/README.md

```diff
@@ -0,0 +1,12 @@
+### Usage
```

Comment:
I think more explanations on the API, summary of perf and accuracy of llama2/llama3 will be helpful, similar to https://github.com/pytorch/ao/tree/main/torchao/prototype/quant_llm

Reply:
Hi @jerryzh168 , I summarized the usage and posted partial results at [README.md](https://github.com/yiliu30/torchao-fork/blob/auto_round_support-3/torchao/prototype/autoround/README.md). More tests are WIP.